.github/workflows/subql_deploy_workflow.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
           node-version: 18
           cache: npm
       - name: Setup Yarn
-        run: yarn install --frozen-lockfile
```

Comment:
Why? 🤔

Reply:
We need to add the package `abab` in order to index evm blocks, as suggested by ji from subquery. See this related [issue](https://github.com/subquery/subql/issues/1277 )